velox/type/Timestamp.h

```diff
@@ -32,6 +32,7 @@ namespace facebook::velox {
 
 struct Timestamp {
  public:
+  enum Precision { Milliseconds = 3, Nanoseconds = 9 };
```

Comment:
- Use enum class
- naming: kMilliseconds, kNanoseconds

See https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md for more style guidelines.